services/estafette/service.go

```diff
@@ -104,7 +104,7 @@ func (s *service) CreateBuild(ctx context.Context, build contracts.Build, waitFo
 
 	// inject build stages
 	if hasValidManifest {
-		mft, err = api.InjectSteps(s.config.ManifestPreferences, mft, builderTrack, shortRepoSource, s.supportsBuildStatus(build.RepoSource))
+		mft, err = api.InjectSteps(s.config.ManifestPreferences, s.config.Integrations.SnykIO, mft, builderTrack, shortRepoSource, s.supportsBuildStatus(build.RepoSource))
```

Comment:
I think we should also have a config for choosing whether you want the snyk step or not